libs/SalesforceHybrid/res/xml/config.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <widget xmlns     = "http://www.w3.org/ns/widgets"
```

Comment:
<table data-meta="generated_by_SalesforceHybrid">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/d5cdf12e048a3cd2c538de73efe4d6343613257e/libs/SalesforceHybrid/res/xml/config.xml#L2"></span>The resource <code>R.xml.config</code> appears to be unused</td>
    </tr>
  </tbody>
</table>